torch/nn/parallel/scatter_gather.py

```diff
@@ -54,8 +54,8 @@ def scatter_kwargs(inputs, kwargs, target_gpus, dim=0):
 
 def gather(outputs, target_device, dim=0):
     r"""
-    Gathers tensors from different GPUs on a specified device
-      (-1 means the CPU).
+    Gathers tensors from different GPUs on a specified device.
+    (Use 'cpu' for CPU)
```

Comment:
Nit: Might be good to add a comment about the deprecation warning you encountered to explain this for future readers